### DISCUSSION TITLE
panel brandname should accept closure

### DISCUSSION THREAD

**Discussion by demobiel** [filamentphp/filament#7871] [Ideas]
not sure if this is a real thing you want if so I want to do a pull request if wanted.

for us it would be nice to be able to do: 
```
$panel->brandName( fn ($panel) => $panel->getTenant()->name )
```

**Comment by danharrin** [comment]
Yeah please PR!

**Reply by demobiel** [reply]
I just pushed a pull request.

**Reply by bilogic** [reply]
thanks @demobiel !!

**Reply by demobiel** [reply]
here it works, but I am not running on the latest version of filament.

`            ->brandName(fn() => Filament::getTenant()->customer->name)
`

**Reply by bilogic** [reply]
Yes, my bad. It does, I was changing way too many things at once. :)
